client/a8c-for-agencies/sections/agency-tier/lib/get-tier-benefits.ts

```diff
@@ -21,28 +21,28 @@ const getTierBenefits = ( translate: ( key: string ) => string ): Benefit[] => [
 			'emerging-partner',
 			'agency-partner',
 			'pro-agency-partner',
-			'strategic-agency-partner',
+			'premier-partner',
 		],
 	},
 	{
 		title: translate( 'Earning opportunities' ),
 		description: translate(
-			"All partners get access to referral and reseller earning opportunities across all of Automattic's suite of products."
+			'All partners get access to referral and reseller earning opportunities across all of Automattic’s suite of products. Premier partners receive the highest earning opportunities.'
```

Comment:
We should use `'` instead of `’`
```suggestion
			"All partners get access to referral and reseller earning opportunities across all of Automattic's suite of products. Premier partners receive the highest earning opportunities."
```

Reply:
Thanks! Should be now fixed 👍